cpp/compile.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 set -eu
+set -x
```

Comment:
Do we need this? I think some information will be dumped out in below code. There are some "echos" to show the configs.